WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Add New Package/ShippingLabelAddNewPackage.swift

```diff
@@ -1,32 +1,31 @@
 import SwiftUI
 
 struct ShippingLabelAddNewPackage: View {
-    @ObservedObject private var viewModel = ShippingLabelAddNewPackageViewModel()
+    @ObservedObject private var viewModel: ShippingLabelAddNewPackageViewModel
 
     var body: some View {
-        VStack(spacing: 0) {
-            VStack(spacing: 0) {
-                SegmentedView(selection: $viewModel.selectedIndex, views: [Text(Localization.customPackage), Text(Localization.servicePackage)])
-                    .frame(height: 44)
-                Divider()
-            }.hidden()
-            ScrollView {
-            }
+        ScrollView {
+            let servicePackagesViewModel = ShippingLabelPackageListViewModel(state: viewModel.state, packagesResponse: viewModel.packagesResponse)
+            ShippingLabelPackageList(viewModel: servicePackagesViewModel)
+                .background(Color(.systemBackground))
         }
+        .background(Color(.listBackground))
         .navigationBarTitle(Text(Localization.title), displayMode: .inline)
     }
+
+    init(siteID: Int64) {
+        viewModel = ShippingLabelAddNewPackageViewModel(siteID: siteID)
+    }
 }
 
 private extension ShippingLabelAddNewPackage {
     enum Localization {
         static let title = NSLocalizedString("Add New Package", comment: "Add New Package screen title in Shipping Label flow")
-        static let customPackage = NSLocalizedString("Custom Package", comment: "Custom Package menu in Shipping Label Add New Package flow")
-        static let servicePackage = NSLocalizedString("Service Package", comment: "Service Package menu in Shipping Label Add New Package flow")
     }
 }
 
 struct ShippingLabelAddNewPackage_Previews: PreviewProvider {
     static var previews: some View {
-        ShippingLabelAddNewPackage()
+        ShippingLabelAddNewPackage(siteID: 1234)
```

Comment:
I can't think of a straightforward way to do this, but do you think it'd be possible to fake the site in a way that will make this SwiftUI preview work? As it is, the preview just shows a blank screen. It isn't critical, but it'd be nice to have the preview working.

Reply:
I thought about this thing, but this would involve providing a method to access data from the outside (therefore without going through the Yosemite layer), and which would be used only for the preview, and I didn't particularly like the solution. 🤔